addons/event_sale/tests/test_event_sale.py

```diff
@@ -87,15 +92,17 @@ def test_event_crm_sale(self):
                     'event_ticket_id': ticket1.id,
                     'product_id': ticket1.product_id.id,
                     'product_uom_qty': TICKET1_COUNT,
-                    'price_unit': 10,
+                    'price_unit': ticket1.price,
                 }), (0, 0, {
                     'event_id': self.event_0.id,
                     'event_ticket_id': ticket2.id,
                     'product_id': ticket2.product_id.id,
                     'product_uom_qty': TICKET2_COUNT,
-                    'price_unit': 50,
+                    'price_unit': ticket2.price,
                 })
-            ]
+            ],
+            # Override the default price list with an empty one
+            'pricelist_id': self.env['product.pricelist'].sudo().create({'name': 'empty_pricelist'}).id,
```

Comment:
Can't that empty price list be defined in setUp and used in the different tests, modified as necessary?

Reply:
Yes I'have reworked the test following this idea and it is easier.